### PR TITLE
fix: Convert js files from es6 to es5

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #588

## Summary

Make these files run in es5

### CODE REVIEW

File: src/demos/head.js [hunk 1]
```diff
@@ -11,9 +11,11 @@
   function loadScript(url, options) {
     let script = document.createElement("script");
     if (options) {
-      Object.keys(options).forEach((key) => {
-        script[key] = options[key];
-      });
+      var keys = Object.keys(options);
```
Comment (jcfranco) [inline src/demos/head.js]: `forEach` should work in ES5.

File: src/demos/demoPageReloader.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
-(function() {
+(function () {
   // based on pieces of https://github.com/ionic-team/stencil/blob/4960fbe11444a18def3ce48ab46b018c416f9472/src/dev-server/dev-client/client-web-socket.ts
+  var getSocketUrl = function getSocketUrl(location) {
+    return "".concat(location.protocol === "https:" ? "wss:" : "ws:", "//").concat(location.hostname, ":").concat(location.port, "/");
```
Comment (jcfranco) [inline src/demos/demoPageReloader.js]: Why not use `+` vs concat?

File: src/demos/demoPageReloader.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
-(function() {
+(function () {
   // based on pieces of https://github.com/ionic-team/stencil/blob/4960fbe11444a18def3ce48ab46b018c416f9472/src/dev-server/dev-client/client-web-socket.ts
+  var getSocketUrl = function getSocketUrl(location) {
```
Comment (jcfranco) [inline src/demos/demoPageReloader.js]: `const` should work in ES5.

File: src/demos/head.js (hunk 1 above)
Comment (driskull) [reply src/demos/head.js]: hmm this was throwing an error in IE11

File: src/demos/demoPageReloader.js [hunk 4]
```diff
@@ -1,12 +1,11 @@
...
-  const getSocketUrl = (location) =>
-    `${location.protocol === "https:" ? `wss:` : `ws:`}//${location.hostname}:${location.port}/`;
+  const getSocketUrl = function getSocketUrl(location) {
+    return location.protocol === "https:" ? "wss:" : "ws:" + "//" + location.hostname + ":" + location.port + "/";
+  };
 
   const clientWebSocket = new window.WebSocket(getSocketUrl(window.location), ["xmpp"]);
-
-  clientWebSocket.addEventListener("message", (message) => {
+  clientWebSocket.addEventListener("message", function(message) {
```
Comment (jcfranco) [inline src/demos/demoPageReloader.js]: Nitpick: can you restore the newline here?

**Pull request merged**